src/tests/ReporterResourcesTest.java

```diff
@@ -0,0 +1,32 @@
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import gov.nasa.jpf.Config;
+import gov.nasa.jpf.JPF;
+import gov.nasa.jpf.util.test.TestJPF;
+
+/**
+ * This is a plain JUnit test to check whether required resource files exist on
+ * JPF classpath.
+ *
+ * @author Jeanderson Candido
+ *
+ */
+public class ReporterResourcesTest extends TestJPF {
```

Comment:
Although it wasn't required, I extended TestJPF to keep this working on the Ant build.
If you have any suggestions, please, let me know so I can update this pull request with your suggestions.